src/Symfony/Component/Messenger/Command/AbstractFailedMessagesCommand.php

```diff
@@ -147,9 +153,9 @@ protected function printPendingMessagesMessage(ReceiverInterface $receiver, Symf
         }
     }
 
-    protected function getReceiver(): ReceiverInterface
+    protected function getReceiver(string $name): ReceiverInterface
```

Comment:
The new argument is a BC break also :p. If the user sub-classes this, and currently has `protected function getReceiver()` with no args, suddenly their signature is wrong.

The fix is... pretty nuts :). Here is an example: https://github.com/symfony/symfony/pull/31249/files#diff-c9a47747d9fe6a39e79059518d9216e550ee0b0723802330a50438f36013c5fbL201

Reply:
Hmm, I think we need some error checking here. What if a bogus `$name` is passed? We should check `if (!$this->failureTransports->has($name))` and throw an exception if it's not found (this would cover people passing us a bad `--failure-transport` option.